pkg/cmd/templates/templater.go

```diff
@@ -33,10 +34,25 @@ func ActsAsRootCommand(cmd *cobra.Command, filters []string, groups ...CommandGr
 	return templater
 }
 
+func mainUsageTemplate() string {
```

Comment:
This looks pretty similar to [ktemplates.MainUsageTemplate](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/templates/templates.go#L77), any reason to re-define it on our codebase?